scripts/ensure.sh

```diff
@@ -13,6 +13,3 @@ fi
 # Install the Node dependencies for the website and the infrastructure.
 yarn install
 yarn --cwd infrastructure install
-
-# Fetch Hugo modules.
-hugo mod vendor
```

Comment:
One other thing for my understanding--what is the benefit of vendoring if it's essentially optional in the build process?